packages/router-bridge/src/router/index.js

```diff
@@ -26,7 +26,7 @@ try {
 	useParams = reactRouterV5.useParams;
 	useRouteMatch = reactRouterV5.useRouteMatch;
 
-	history = createBrowserHistory();
+	history = createBrowserHistory({ basename: window.basename || '/' });
```

Comment:
For app on RR v5, we need to pass the basename at history creation.
To avoid breaking change, history is still being created in router bridge at import of the file. The app has to set basename var in the js entry point to allow router bridge to set it in history.